docs/arf.md

```diff
@@ -13,6 +13,8 @@ technical Architecture and Reference Framework (hereinafter the ARF), a
 set of common standards and technical specifications and a set of common
 guidelines and best practices.
 
+This is a sample addition demonstrating how to propose changes with RPs.
```

Comment:
An example of how to comment or initiate a discussion thread regarding a particular change.